src/apiProvider/index.js

```diff
@@ -0,0 +1,29 @@
+import axios from 'axios';
```

Comment:
Just a comment on the location. Did we agree to put this is a `helpers` folder? or is there a reason it belongs better in the root?